### PR TITLE
Start entire app using `npm start`

### PR DESCRIPTION
**Pull request by itaisteinherz**

### CODE REVIEW

**Comment by itaisteinherz** [comment]
Let me know if you think I should remove the documentation about the individual commands and merge it into the new documentation (about running everything concurrently) or keep things the way they are.

**Comment by itaisteinherz** [comment]
Sidenote: when running `$ yarn start:app`, I get the following error:

```
❯ yarn start
yarn run v1.16.0
$ react-scripts start

There might be a problem with the project dependency tree.
It is likely not a bug in Create React App, but something you need to fix locally.

The react-scripts package provided by Create React App requires a dependency:

  "webpack": "4.29.6"

Don't try to install it manually: your package manager does it automatically.
However, a different version of webpack was detected higher up in the tree:

  /create-react-app-lambda/node_modules/webpack (version: 4.19.1)

Manually installing incompatible versions is known to cause hard-to-debug issues.

If you would prefer to ignore this check, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That will permanently disable this message but you might encounter other issues.

To fix the dependency tree, try following the steps below in the exact order:

  1. Delete package-lock.json (not package.json!) and/or yarn.lock in your project folder.
  2. Delete node_modules in your project folder.
  3. Remove "webpack" from dependencies and/or devDependencies in the package.json file in your project folder.
  4. Run npm install or yarn, depending on the package manager you use.

In most cases, this should be enough to fix the problem.
If this has not helped, there are a few other things you can try:

  5. If you used npm, install yarn (http://yarnpkg.com/) and repeat the above steps with it instead.
     This may help because npm has known issues with package hoisting which may get resolved in future versions.

  6. Check if /create-react-app-lambda/node_modules/webpack is outside your project directory.
     For example, you might have accidentally installed something in your home folder.

  7. Try running npm ls webpack in your project folder.
     This will tell you which other package (apart from the expected react-scripts) installed webpack.

If nothing else helps, add SKIP_PREFLIGHT_CHECK=true to an .env file in your project.
That would permanently disable this preflight check in case you want to proceed anyway.

P.S. We know this message is long but please read the steps above :-) We hope you find them helpful!

error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

And so I uninstalled `babel-loader`, which seemed to fix the issue. Should I remove the package from here as well (either in this PR or in a separate one)?

**Comment by sw-yx** [comment]
ok could you remove babel loader too? thanks

**Comment by itaisteinherz** [comment]
@sw-yx Done 👌🏻
Will users still be able to use [function builder detection](https://www.netlify.com/blog/2019/04/24/zero-config-yet-technology-agnostic-how-netlify-dev-detectors-work/) despite the `start` npm script now running both the frontend app and `netlify-lambda`? From the docs, it seems like the feature should still work, though now they'll have to set `command="yarn start:app"` in `netlify.toml`.

File: package.json [hunk 1]
```diff
@@ -9,7 +9,8 @@
     "react-scripts": "^3.0.0"
   },
   "scripts": {
-    "start": "react-scripts start",
+    "start": "run-p start:**",
```
[the remark this answers is not in the document]
Comment (itaisteinherz) [reply package.json]: Still relevant? (As you merged the PR.)
If so, let me know and I'll open a new one with the modification.

[the remark this answers is not in the document]
Comment (sw-yx) [reply package.json]: nah

**Pull request merged**